lib/ws_servers/algos/handlers/on_open.js

```diff
@@ -62,6 +63,21 @@ module.exports = async (server, ws, msg) => {
     server.broadcast(userID, ['started', userID, exID, name, label, gid, args])
   })
 
+  hosts[key].on('meta:reload', async() => {
+    const { AlgoOrder } = algoDB
+
+    const aos = await AlgoOrder.find([['active', '=', true]])
+    
+    if (aos.length === 0) {
+      return
+    }
+
+    // aoc.userID = userID
+    // await aoc.loadAOs('bitfinex', aos)
```

Comment:
hm, server takes the client and calls itself... not a good idea...

Reply:
how about broadcasting to the algo client, like in other cases too. 

another layer of indirection we might want to delete in the future, but follows the client/server model

Reply:
1. create a handler in ws_clients/algos, e.g. for the rpc command "foo"
2. broadcast: `server.broadcast(userID, ['foo', userID])`
3. in the algo orders client, find active orders and run the command